CHANGELOG.md

```diff
@@ -3,6 +3,103 @@ FEATURES:
 * CLI
   * The `consul-k8s` CLI enables users to deploy and operate Consul on Kubernetes.
     * Support `consul-k8s install` command. [[GH-713](https://github.com/hashicorp/consul-k8s/pull/713)]
+* Helm Chart
+  * Add support for Admin Partitions. **(Consul Enterprise only)**
+  **ALPHA** [[GH-729](https://github.com/hashicorp/consul-k8s/pull/729)]
+
+This feature allows Consul to be deployed across multiple Kubernetes clusters while sharing a single set of Consul
+servers. The services on each cluster can be independently managed. This feature in the alpha phase requires a flat pod
+and node network in order for inter-partition networking to work. This feature requires TLS to be enabled. This feature
```

Comment:
why is TLS required?

Reply:
We require HTTPS for communication to externalServers. We had that already as a requirement and I decided not to loosen it for AP.